page-030.tex

```diff
@@ -48,7 +48,7 @@
 
 It sometimes happens that one has passed with the point, and thrust
 that the Enemy in that moment pulls his Arm back and [laederar???]; In
-this it the [faut???] of the one who passed, taht he did not Continued
+this it the [faut???] of the one who passed, that he did not Continued
 to his Enemy's body, or did not take tempo well; Only if he in the
 same, when the Enemy moves the Blade in front of him, or in that he is
 with [deffeta] occupied, or goes out of presenza, he cannot at the
```

Comment:
presenza: Will we use "presence", here, too?

Reply:
Yes, I think it may be worth adding to the dictionary as well (that's what I have been using to try to get a consistent translation, although not always succeeding).

Reply:
Added to dict.tex in https://github.com/vatine/palaestra-svecana/pull/34